spyder/plugins/ipythonconsole/widgets/figurebrowser.py

```diff
@@ -60,12 +60,13 @@ def _handle_display_data(self, msg):
             if (self.figurebrowser is not None and
                     self.figurebrowser.mute_inline_plotting):
                 if not self.sended_render_message:
-                    msg['content']['data']['text/plain'] = self._append_html(
+                    msg['content']['data']['text/plain'] = ''
+                    self._append_html(
                         _('<hr>'
                           '\nFigures now render in the Plots pane by default. '
                           'To make them also appear inline in the Console, '
                           'uncheck "Mute Inline Plotting" under the Plots '
-                          'pane options menu. \n<hr>'))
+                          'pane options menu. \n<hr><p></p>'))
```

Comment:
I don't like this `<p></p>` you added here. Could you change it for a `<br>` instead? Also, please add `<br>` before the first `<hr>` above, for consistency. Thanks!

Reply:
I am sorry. I made mistakes because I was not used to git yet. Thank you for reopening it.

Reply:
No problem :+1: